gui/screens/settings.py

```diff
@@ -237,12 +237,18 @@ def __init__(self):
         self._wallet_networks_item = QObject('settingsContentBaseScrollView_WalletNetworkDelegate')
         self._testnet_text_item = QObject('settingsContentBaseScrollView_Goerli_testnet_active_StatusBaseText')
         self._testnet_mode_button = Button('settings_Wallet_NetworksView_TestNet_Toggle')
+        self._testnet_mode_title = TextLabel('settings_Wallet_NetworksView_TestNet_Toggle_Title')
+        self._back_button = Button('main_toolBar_back_button')
```

Comment:
Were you planning to add button to the class only or maybe we can add verification that button is present? Like it was done in test_change_account_order_not_possible, for example ?

Reply:
good point, i was planning indeed but forgot :D